packages/hs-react-ui/src/components/Button/ButtonContainers.tsx

```diff
@@ -1,7 +1,96 @@
-import styled from 'styled-components';
-import { darken } from 'polished';
-import { ButtonTypes, ButtonContainer } from './Button';
+import styled, { StyledComponentBase, css, keyframes } from 'styled-components';
+import { darken, getContrast, complement } from 'polished';
 import colors from '../../enums/colors';
+import fonts from '../../enums/fonts';
+
+type StyledButtonProps = {
+  elevation: number;
+  color?: string;
+  fillColor?: string;
+};
+export enum ButtonTypes {
+  primary = 'primary',
+  secondary = 'secondary',
+  default = 'default',
+  destructive = 'destructive',
+  link = 'link',
+  outline = 'outline',
+}
```

Comment:
I think these should just be
- fill
- link
- outline

where primary, secondary, default, and destructive, are just accessable through choosing colors